content/articles/using-multiple-ssh-keys-for-multiple-github-accounts/index.md

```diff
@@ -0,0 +1,129 @@
+For most developers, there is a need to run multiple GitHub accounts on one computer. For instance, You can have an organization's GitHub account and another for your projects at home.
+  
+In this article, you will learn how to [use multiple SSH keys for different GitHub accounts](https://gist.github.com/jexchan/2351996). While working with two different GitHub accounts, you must set them up using an SSH key. Then you need to set up SSH config to manage multiple GitHub accounts on one computer.
+
+### Table of contents
+[What are SSH Keys](#what-are-ssh-keys)
+[How to manage SSH Keys on GitHub Accounts](#how-to-manage-ssh-keys-on-github-accounts)
+[Creating and updating the SSH config file](#creating-and-updating-the-ssh-config-file)
+[Cloning the Repositories](#cloning-the-repositories)
+
+### Prerequisites
+To grasp what this article entails. It is crucial to have a good understanding of how GitHub works.
+
+### What are SSH Keys
+[SSH refers to Secure Shell](https://en.wikipedia.org/wiki/Secure_Shell_Protocol). A cryptographic network protocol that allows a single computer to connect to a server over the internet securely. SSH is best used for accessing remote servers.
+
+SSH is designed to offer secure encryption, verification, and communication between computers. It provides a safe way of executing commands and configuring services remotely. The SSH technology is often performed using the client-server model. So you can securely log in remotely from one system to another and transmit files.
```

Comment:
`client-server model`

Add 1 - 2 lines of explanation about it, and reference a link for more info about it.